.github/workflows/docker.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2024 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# GitHub Actions workflow to publish Docker images.
+name: "Docker"
+on:
+  push:
```

Comment:
I think we could limit this to on push to the `main` branch.

Reply:
yeah, I am "publish" happy, I will update 👍